portfolio/src/main/webapp/script.js

```diff
@@ -73,11 +73,11 @@ function contact() {
 
 function getComments() {
 	fetch('/data').then(response => response.json()).then((comments) => {
-		console.log(comments);
 		const bodyElement = document.body;
-		bodyElement.innerHTML += 'Here are the comments: \n';
-        for (comment of comments) {
-            bodyElement.innerHTML += comment + '\n';
+        console.log('Comments: ', comments);
+        bodyElement.innerHTML += 'Here are all the comments: <br>';
+        for (index in comments) {
```

Comment:
nit - use a `for...of` here to improve readability
```suggestion
        for (comment of comments) {
```